src/core/metallicroughnesseffect.cpp

```diff
@@ -205,14 +205,14 @@ MetallicRoughnessEffect::MetallicRoughnessEffect(Qt3DCore::QNode *parent)
     , m_metalRoughES3Shader(new QShaderProgram(this))
     , m_metalRoughES2Shader(new QShaderProgram(this))
 {
-    const auto enabledLayers = QStringList{ QStringLiteral("noBaseColorMap"),
-                                            QStringLiteral("noMetalRoughMap"),
-                                            QStringLiteral("noAmbientOcclusionMap"),
-                                            QStringLiteral("noEmissiveMap"),
-                                            QStringLiteral("noNormalMap"),
-                                            QStringLiteral("noHasColorAttr"),
-                                            QStringLiteral("noDoubleSided"),
-                                            QStringLiteral("noHasAlphaCutoff") };
+    const auto enabledLayers = QStringList { QStringLiteral("noBaseColorMap"),
```

Comment:
Hm... this was done this way by clang-format. Are you sure about that format change?